src/Forms/Main.cs

```diff
@@ -13726,7 +13726,7 @@ private void toolStripMenuItemAdjustAllTimes_Click(object sender, EventArgs e)
             _showEarlierOrLater = new ShowEarlierLater();
             if (!_showEarlierOrLater.IsPositionAndSizeSaved)
             {
-                _showEarlierOrLater.Top = Top + 100;
+                _showEarlierOrLater.Top += 100;

```

Comment:
This doesn't look right: if the position is unknown (`!_showEarlierOrLater.IsPositionAndSizeSaved`) it is determined relative to the position of the Main form (`this.Top` & `this.Left`).